runtime/compiler/control/rossa.cpp

```diff
@@ -1498,6 +1499,16 @@ onLoadInternal(
       }
 #endif
 
+   // enable TR_SelfTuningScratchMemoryUsageBeforeCompile if there is no swap memory
+   J9MemoryInfo memInfo;
```

Comment:
Maybe this code should stay in the early processing of options (fePreProcess) since it sets an option